src/NextGen/DonationForm/ViewModels/DonationFormViewModel.php

```diff
@@ -117,7 +117,9 @@ public function exports(): array
 
         return [
             'donateUrl' => $donateUrl,
-            'successUrl' => give_get_success_page_uri(),
+            'inlineRedirectRoutes' => [
```

Comment:
Where is this being used? Also, it seems odd to be exporting a hard-coded value here.

Reply:
I see the reference in `app/form/Form.tsx`, but what is the reason for localizing this hard-coded value?

Reply:
Oh yeah, my thought was there will be be more inline routes added here that will come from some kind of registrar.  Another example will be a form error route.  I want the front-end to be aware that these routes exist but have the backend supply them.  The registrar will likely tie into the the new Route facade.